### DISCUSSION TITLE
Javascript const/;

### DISCUSSION THREAD

**Discussion by NellyisDevv** [community/community#20689] [Projects and Issues] [answered]
I had to reset my VS Code and on reset a lot of things changed to default, one thing I really want to get back is the red lettering next to "const" in my old setup the credits variable would be red, also something that I don't see everywhere is the symbol at the end of a variable or other statements, any help is appreciated, thanks!

<img width="369" alt="Screen Shot 2022-07-16 at 12 35 22 PM" src="https://user-images.githubusercontent.com/100225556/179363957-3d60be24-83c2-448b-84ae-4d66bb544857.png">

**Comment by nikhilstacks** [comment]
You should download back the theme you had previously.

**Comment by thuongtruong1009** [comment, marked as answer]
Hi @NellyisDevv 
You have two options:
* First, you log in to VS Code with your old account which has been login before the reset, and all settings will be back up. It useful if you sync all settings with Github before
* Second, I think this theme is [Onedark theme](https://marketplace.visualstudio.com/items?itemName=akamud.vscode-theme-onedark). You also can search [Dracule Theme](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula) which I using.
* Tip: Use [setting sync extension](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync) to backup and sync all your settings to GitHub.

**Comment by Rumata-SL** [comment]
Сan be changed in json.
